modules/gltf/src/lib/types/gltf-ext-mesh-features-schema.ts

```diff
@@ -0,0 +1,53 @@
+import {GLTFTextureInfoMetadata} from './gltf-json-schema';
```

Comment:
If we are adding lots of schemas, wonder if we should create a folder `src/lib/types/schemas/`?